packages/lesswrong/lib/collections/posts/formGroups.ts

```diff
@@ -90,7 +90,7 @@ export const formGroups: Partial<Record<string,FormGroupType>> = {
     startCollapsed: true
   },
   tags: {
-    order: isEAForum ? 22 : 60,
+    order: isEAForum ? 20 : 60,
```

Comment:
What's the change here for?

Reply:
This makes "Set topics" the first group under the post body in the form